files/package.json

```diff
@@ -49,8 +49,10 @@
     "@glint/environment-ember-loose": "^1.4.0",
     "@glint/environment-ember-template-imports": "^1.4.0",
     "@glint/template": "^1.4.0",
+    "@ember/app-tsconfig": "^1.0.0",
```

Comment:
used for the main tsconfig / test app

library tsconfig is now only for publish